monerochad/commands/xmr_chart.py

```diff
@@ -24,41 +25,41 @@
 	app_commands.Choice(name="15 days", value="21600;15d"),
 ]
 
-# Remember to update the command description too when changing the cooldown
-COOLDOWN_SECONDS = 15
-
-prevtime = 0.0
 
 def register(group: app_commands.Group):
 	
 	marketcolors = mpf.make_marketcolors(base_mpf_style="yahoo", wick="inherit")
 	mpf_style = mpf.make_mpf_style(base_mpf_style="yahoo", marketcolors=marketcolors)
-	
+
 	@group.command(name="chart", description="Get Monero price chart (has 15 second cooldown)")
 	@app_commands.describe(vs_val="Symbol of the coin to compare XMR to", interval_val="How much time should one candle represent")
 	@app_commands.choices(vs_val=VS_CHOICES, interval_val=INTERVAL_CHOICES)
 	@app_commands.rename(vs_val="vs", interval_val="interval")
+	@commands.cooldown(1, 15, commands.BucketType.user) ##
```

Comment:
I think we should keep how I've made it since it doesn't really make too much of a difference as I suppose this bot is only in the monero server and is used by like only 5 people at max (nk, mikejacko, me...etc)